Makefile

```diff
@@ -54,6 +54,19 @@ docker-build: test
 docker-push:
```

Comment:
docker image can also be automatically generated by GitHub  action 
https://github.com/oam-dev/rudr/blob/master/.github/workflows/dev.yml